elide-model-config/src/test/resources/validator/duplicate_security_role/models/security.hjson

```diff
@@ -0,0 +1,6 @@
+{
+    roles:  [
+        Admin User
+        prefab.role.all
```

Comment:
I want to talk about this test.  I'm a bit confused here about prefab.role.all.  Are we trying to define this is a role or reference the "Prefab.Role.All" that exists as prefabricated security role?

I assume the former, but I'm not sure what the goal of this is.  This looks like something nobody should ever try to do.

Reply:
this is negative test to ensure validation fails if someone tries to define the already defined roles. Role "Prefab.Role.All" already exists so validation must complain.